books/RayTracingTheNextWeek.html

```diff
@@ -2057,105 +2057,147 @@
 
 Storing Texture Image Data
 ---------------------------
-Now we also need to create a texture class that holds an image. I am going to use my favorite image
-utility [stb_image][]. It reads in an image into a big array of unsigned char. These are just packed
-RGBs with each component in the range [0,255] (black to full white). The `image_texture` class uses
-the resulting image data.
+Now it's time to create a texture class that holds an image. I am going to use my favorite image
+utility, [stb_image][]. It reads image data into a big array of unsigned char. These are just packed
+RGBs with each component in the range [0,255] (black to full white). To help make loading our image
+files even easier, we provide a helper class to manage all this -- `rtw_image`:
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
-    #include "rtweekend.h"
-    #include "rtw_stb_image.h"
-    #include "perlin.h"
-
-    #include <iostream>
-
-    ...
-
-    class image_texture : public texture {
-      public:
-        const static int bytes_per_pixel = 3;
+    #ifndef RTWEEKEND_STB_IMAGE_H
+    #define RTWEEKEND_STB_IMAGE_H
 
-        image_texture() : data(nullptr), width(0), height(0), bytes_per_scanline(0) {}
+    // Disable pedantic warnings for this header from the Microsoft Visual C++ compiler.
```

Comment:
I understand the pull of snark, but consider removing "pedantic"

Reply:
Actually wasn't being snarky here, but it looks like "pedantic" is more exclusively pejorative than I'd realized. Will fix.

Reply:
I get that you're using "pedantic" as a strictly technical term here, but it sounds like snark to a lay audience.